app/src/main/java/tttTDD/Java/Display.java

```diff
@@ -0,0 +1,87 @@
+package tttTDD.Java;
+
+import java.util.ArrayList;
+import java.util.stream.IntStream;
+
+public class Display {
+
+    public void render(String message) {
+        System.out.println(message);
+    }
+
+    public String playerSelectionMessage() {
+        return "What type of player is this?[Enter a number]\n 1. Human\n 2. Computer\n 3. Random Computer\n";
+    }
+
+    public String negativeBoardSizeMessage() {
+        return "Can't have the Universe implode on us. Try a larger number:";
+    }
+
+    public String wrongTypeMessage() {
+        return "Stop it. Enter a number as an input:";
+    }
+
+    public String tryAgainMessage() {
+        return "Oops... incorrect move. Please try again";
+    }
+
+    public String enterBoardSizeMessage() {
+        return "Please enter a board size number:";
+    }
+
+    public String makeMoveMessage() {
+        return "Make your move:";
+    }
+
+    public String showMovePerformedMessage(Game game, int value) {
+        String marker = game.getCurrentPlayer().getMarker().equals(game.getPlayer1Marker()) ? game.getPlayer2Marker() : game.getPlayer1Marker();
+        return String.format("Player %s has made a move on space %s", marker, value);
+    }
+
+    public String renderBoard(Game game, int boardSize) {
```

Comment:
this method name is a bit deceptive, since the other `render` method on this class is a void that prints to stdout.

Reply:
bump